src/components/tooltip/index.js

```diff
@@ -30,12 +30,12 @@ const TooltipNode = styled('div')`
   background-color: ${props => props['background-color']};
 `
 
-const TooltipLabel = styled('strong')(({ typography = typographyDefaultProps }) =>`
+const TooltipLabel = styled('strong')(({ typography = typographyDefaultProps.typographyProps }) =>`
```

Comment:
Minor: I guess you don't need to do ` typography = typographyDefaultProps.typographyProps ` since you already declared the default for typography at the end of the file. So you can do just `typography` and that will work just fine

Reply:
I think this was a recommendation from a previous PR (correct me if I'm wrong, @geoerika).

The idea was that `TooltipLabel` should be treated as a pure function and should handle its own defaults. Then you could easily pull it out and use it outside of this context.

Reply:
This I discussed with Leo extensively last week: https://eqworks.slack.com/archives/GV827LS0M/p1600449195002500?thread_ts=1600305645.009600&cid=GV827LS0M

Reply:
Yes Shane! Thanks!